control/Director.php

```diff
@@ -841,6 +841,7 @@ protected static function force_redirect($destURL) {
 	 * @return boolean|string String of URL when unit tests running, boolean FALSE if patterns don't match request URI
 	 */
 	public static function forceSSL($patterns = null, $secureDomain = null) {
+		if(Director::is_cli()) return false;
```

Comment:
This needs to return the $destURL when running tests. You need to make return false ONLY if not running tests.